Shared/Tests.Shared/TestObjects.cs

```diff
@@ -209,6 +210,25 @@ public class Owner : RealmObject
     }
 
     [SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1402:FileMayOnlyContainASingleClass")]
+    public class Employee : RealmObject
+    {
+        public string Name { get; set; }
+
+        public IList<Employee> Reports { get; }
+
+        [Backlink(nameof(Employee.Reports))]
+        public IQueryable<Employee> Bosses { get; }
+
+        public Employee Boss
+        {
+            get
+            {
+                return Bosses.ToList().SingleOrDefault();
```

Comment:
@nirinchev I'm just novice user to c# and unit test were the first I'd go for when I didn't google any info. #1084 I have no strong opinion on this, I want just add that users may take a look at tests when they are desperate. Generally, 👍  for putting info about workarounds to docs!